Homework2/TrieTests/TrieTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Trie;
+using System;
+
+namespace Trie.Tests
+{
+    [TestClass()]
```

Comment:
Скобки в атрибутах можно не писать:
```suggestion
    [TestClass]
```